adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDDFunctions.scala

```diff
@@ -49,6 +51,65 @@ class NucleotideContigFragmentRDDFunctions(rdd: RDD[NucleotideContigFragment]) e
   }
 
   /**
+   * Save nucleotide contig fragments in FASTA format.
+   *
+   * @param fileName file name
+   */
+  def adamSaveAsFasta(fileName: String) = {
+
+    def isFragment(record: NucleotideContigFragment): Boolean = {
+      (Option(record.fragmentNumber), Option(record.numberOfFragmentsInContig)) match {
```

Comment:
Also, as an aside, here and elsewhere, you're using `record.fragmentNumber`, where you should be using `record.getFragmentNumber`. While `record.fragmentNumber` is a valid field, access directly to the field should be deprecated, so you should be seeing deprecation warnings on these accesses.